statesync/stateprovider.go

```diff
@@ -96,6 +97,18 @@ func NewLightClientStateProvider(
 	}, nil
 }
 
+func uniqServers(servers []string) []string {
+	serversSet := make(map[string]struct{})
+	for _, server := range servers {
+		serversSet[server] = struct{}{}
+	}
+	servers = []string{}
```

Comment:
NITS: I think renaming it to uniqServers will prevent confusion with servers.

Reply:
This comment isn't important things and you can "Resolve Conversation" each of them yourself if it doesn't require any particular fix.